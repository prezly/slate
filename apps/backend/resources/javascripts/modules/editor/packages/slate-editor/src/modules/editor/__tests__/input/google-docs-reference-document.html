<!-- https://docs.google.com/document/d/1Dv3YK0qZ0c_Wd8qinvETjTlClrYz9NHQE-TIVIqR99U/edit# -->
<meta http-equiv="content-type" content="text/html; charset=utf-8"><meta charset="utf-8"><b style="font-weight:normal;" id="docs-internal-guid-4fe3b5c4-7fff-ce89-2057-008d2f503451"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:3pt;"><span style="font-size:26pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Title</span></p><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:16pt;"><span style="font-size:15pt;font-family:Arial;color:#666666;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Subtitle</span></p><h1 dir="ltr" style="line-height:1.38;margin-top:20pt;margin-bottom:6pt;"><span style="font-size:20pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Heading 1</span></h1><h2 dir="ltr" style="line-height:1.38;margin-top:18pt;margin-bottom:6pt;"><span style="font-size:16pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Heading 2</span></h2><h3 dir="ltr" style="line-height:1.38;margin-top:16pt;margin-bottom:4pt;"><span style="font-size:13.999999999999998pt;font-family:Arial;color:#434343;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Heading 3</span></h3><h4 dir="ltr" style="line-height:1.38;margin-top:14pt;margin-bottom:4pt;"><span style="font-size:12pt;font-family:Arial;color:#666666;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Heading 4</span></h4><h5 dir="ltr" style="line-height:1.38;margin-top:12pt;margin-bottom:4pt;"><span style="font-size:11pt;font-family:Arial;color:#666666;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Heading 5</span></h5><h6 dir="ltr" style="line-height:1.38;margin-top:12pt;margin-bottom:4pt;"><span style="font-size:11pt;font-family:Arial;color:#666666;background-color:transparent;font-weight:400;font-style:italic;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Heading 6</span></h6><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Bold </span><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Normal text </span><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:italic;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Italics </span><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:underline;-webkit-text-decoration-skip:none;text-decoration-skip-ink:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Underline </span><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:italic;font-variant:normal;text-decoration:underline;-webkit-text-decoration-skip:none;text-decoration-skip-ink:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Bold italics underline </span><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:line-through;-webkit-text-decoration-skip:none;text-decoration-skip-ink:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Strikethrough </span><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:underline line-through;-webkit-text-decoration-skip:none;text-decoration-skip-ink:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Strikethrough underline </span><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">superscript</span><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span style="font-size:0.6em;vertical-align:super;">2 </span></span><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">subscript</span><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span style="font-size:0.6em;vertical-align:sub;">2 </span></span><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">normal text, </span><a href="https://prezly.com" style="text-decoration:none;"><span style="font-size:11pt;font-family:Arial;color:#1155cc;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:underline;-webkit-text-decoration-skip:none;text-decoration-skip-ink:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">prezly.com</span></a><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">,</span></p><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span style="border:none;display:inline-block;overflow:hidden;width:624px;height:187px;"><img src="https://lh6.googleusercontent.com/gEBlS3Y4XfULtVHhUImChzVLc_SjPk6btSO07luV5DQOmyc1MGuejYnxIJsk11ehQqOR3zvEZMnle-DrKCugbCunXcA2mSkOy0VuxVxCioQBvwLs5xuMkEdFJasff7QiKxmsYSMv" width="624" height="187" style="margin-left:0px;margin-top:0px;" /></span></span></p><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Feugiat pretium nibh ipsum consequat nisl vel pretium. Viverra mauris in aliquam sem fringilla ut morbi tincidunt augue. Proin libero nunc consequat interdum varius sit amet mattis vulputate. Et netus et malesuada fames ac turpis. Odio facilisis mauris sit amet massa. At risus viverra adipiscing at in. Amet luctus venenatis lectus magna. Sagittis aliquam malesuada bibendum arcu vitae elementum curabitur vitae nunc. Massa placerat duis ultricies lacus sed turpis. Blandit massa enim nec dui nunc mattis enim ut tellus.</span></p><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><a href="https://example.com" style="text-decoration:none;"><span style="font-size:11pt;font-family:Arial;color:#1155cc;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:underline;-webkit-text-decoration-skip:none;text-decoration-skip-ink:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span style="border:none;display:inline-block;overflow:hidden;width:300px;height:300px;"><img src="https://lh5.googleusercontent.com/eYxrHVfNo9J2lBAZTw5pEZFFZg6E8zq7H-0LOik3ZipcvplQ_IfGHZtdTQq_y0ztJt1EKl6soPrwaxvPKs8uDAIlnr4kusdSVqWut4GYAsQtVO6LgT-W-jRU-Qolts-pna4Zp16f" width="300" height="300" style="margin-left:0px;margin-top:0px;" /></span></span></a></p><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><hr /></p><ul style="margin-top:0;margin-bottom:0;"><li dir="ltr" style="list-style-type:disc;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;"><p dir="ltr" style="line-height:1.38;margin-top:12pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Nested Lists A</span></p></li><ul style="margin-top:0;margin-bottom:0;"><li dir="ltr" style="list-style-type:circle;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Nested Lists A1</span></p></li><ol style="margin-top:0;margin-bottom:0;"><li dir="ltr" style="list-style-type:lower-roman;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Nested Lists A1a</span></p></li><li dir="ltr" style="list-style-type:lower-roman;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Nested Lists A1b</span></p></li></ol><li dir="ltr" style="list-style-type:circle;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Nested Lists A2</span></p></li><ol style="margin-top:0;margin-bottom:0;"><li dir="ltr" style="list-style-type:lower-roman;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Nested Lists A2a</span></p></li><li dir="ltr" style="list-style-type:lower-roman;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Nested Lists A2b</span></p></li></ol><li dir="ltr" style="list-style-type:circle;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Nested Lists A3</span></p></li></ul><li dir="ltr" style="list-style-type:disc;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Nested Lists B</span></p></li><ul style="margin-top:0;margin-bottom:0;"><li dir="ltr" style="list-style-type:circle;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Nested Lists B1</span></p></li><ol style="margin-top:0;margin-bottom:0;"><li dir="ltr" style="list-style-type:lower-roman;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Nested Lists B1a</span></p></li><li dir="ltr" style="list-style-type:lower-roman;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Nested Lists B1b</span></p></li></ol><li dir="ltr" style="list-style-type:circle;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Nested Lists B2</span></p></li></ul><li dir="ltr" style="list-style-type:disc;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:12pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Nested Lists C</span></p></li></ul><br /></b>
