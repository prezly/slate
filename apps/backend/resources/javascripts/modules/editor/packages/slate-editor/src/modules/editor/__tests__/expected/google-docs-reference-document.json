[
    {
        "type": "paragraph",
        "children": [
            {
                "text": "Title"
            }
        ]
    },
    {
        "type": "paragraph",
        "children": [
            {
                "text": "Subtitle"
            }
        ]
    },
    {
        "type": "heading-one",
        "children": [
            {
                "text": "Heading 1"
            }
        ]
    },
    {
        "type": "heading-two",
        "children": [
            {
                "text": "Heading 2"
            }
        ]
    },
    {
        "type": "heading-two",
        "children": [
            {
                "text": "Heading 3"
            }
        ]
    },
    {
        "type": "heading-two",
        "children": [
            {
                "text": "Heading 4"
            }
        ]
    },
    {
        "type": "heading-two",
        "children": [
            {
                "text": "Heading 5"
            }
        ]
    },
    {
        "type": "heading-two",
        "children": [
            {
                "text": "Heading 6",
                "italic": true
            }
        ]
    },
    {
        "type": "paragraph",
        "children": [
            {
                "text": "Bold ",
                "bold": true
            },
            {
                "text": "Normal text "
            },
            {
                "text": "Italics ",
                "italic": true
            },
            {
                "text": "Underline ",
                "underlined": true
            },
            {
                "text": "Bold italics underline ",
                "bold": true,
                "italic": true,
                "underlined": true
            },
            {
                "text": "Strikethrough "
            },
            {
                "text": "Strikethrough underline ",
                "underlined": true
            },
            {
                "text": "superscript"
            },
            {
                "text": "2 ",
                "superscript": true
            },
            {
                "text": "subscript"
            },
            {
                "text": "2 ",
                "subscript": true
            },
            {
                "text": "normal text, "
            },
            {
                "children": [
                    {
                        "text": "prezly.com",
                        "underlined": true
                    }
                ],
                "href": "https://prezly.com/",
                "type": "link"
            },
            {
                "text": ","
            }
        ]
    },
    {
        "type": "placeholder:image",
        "children": [
            {
                "text": ""
            }
        ]
    },
    {
        "type": "paragraph",
        "children": [
            {
                "text": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Feugiat pretium nibh ipsum consequat nisl vel pretium. Viverra mauris in aliquam sem fringilla ut morbi tincidunt augue. Proin libero nunc consequat interdum varius sit amet mattis vulputate. Et netus et malesuada fames ac turpis. Odio facilisis mauris sit amet massa. At risus viverra adipiscing at in. Amet luctus venenatis lectus magna. Sagittis aliquam malesuada bibendum arcu vitae elementum curabitur vitae nunc. Massa placerat duis ultricies lacus sed turpis. Blandit massa enim nec dui nunc mattis enim ut tellus."
            }
        ]
    },
    {
        "type": "placeholder:image",
        "children": [
            {
                "text": ""
            }
        ]
    },
    {
        "children": [
            {
                "text": ""
            }
        ],
        "type": "divider"
    },
    {
        "type": "bulleted-list",
        "children": [
            {
                "type": "list-item",
                "children": [
                    {
                        "type": "list-item-text",
                        "children": [
                            {
                                "text": "Nested Lists A"
                            }
                        ]
                    },
                    {
                        "type": "bulleted-list",
                        "children": [
                            {
                                "type": "list-item",
                                "children": [
                                    {
                                        "type": "list-item-text",
                                        "children": [
                                            {
                                                "text": "Nested Lists A1"
                                            }
                                        ]
                                    },
                                    {
                                        "type": "numbered-list",
                                        "children": [
                                            {
                                                "type": "list-item",
                                                "children": [
                                                    {
                                                        "type": "list-item-text",
                                                        "children": [
                                                            {
                                                                "text": "Nested Lists A1a"
                                                            }
                                                        ]
                                                    }
                                                ]
                                            },
                                            {
                                                "type": "list-item",
                                                "children": [
                                                    {
                                                        "type": "list-item-text",
                                                        "children": [
                                                            {
                                                                "text": "Nested Lists A1b"
                                                            }
                                                        ]
                                                    }
                                                ]
                                            }
                                        ]
                                    }
                                ]
                            },
                            {
                                "type": "list-item",
                                "children": [
                                    {
                                        "type": "list-item-text",
                                        "children": [
                                            {
                                                "text": "Nested Lists A2"
                                            }
                                        ]
                                    },
                                    {
                                        "type": "numbered-list",
                                        "children": [
                                            {
                                                "type": "list-item",
                                                "children": [
                                                    {
                                                        "type": "list-item-text",
                                                        "children": [
                                                            {
                                                                "text": "Nested Lists A2a"
                                                            }
                                                        ]
                                                    }
                                                ]
                                            },
                                            {
                                                "type": "list-item",
                                                "children": [
                                                    {
                                                        "type": "list-item-text",
                                                        "children": [
                                                            {
                                                                "text": "Nested Lists A2b"
                                                            }
                                                        ]
                                                    }
                                                ]
                                            }
                                        ]
                                    }
                                ]
                            },
                            {
                                "type": "list-item",
                                "children": [
                                    {
                                        "type": "list-item-text",
                                        "children": [
                                            {
                                                "text": "Nested Lists A3"
                                            }
                                        ]
                                    }
                                ]
                            }
                        ]
                    }
                ]
            },
            {
                "type": "list-item",
                "children": [
                    {
                        "type": "list-item-text",
                        "children": [
                            {
                                "text": "Nested Lists B"
                            }
                        ]
                    },
                    {
                        "type": "bulleted-list",
                        "children": [
                            {
                                "type": "list-item",
                                "children": [
                                    {
                                        "type": "list-item-text",
                                        "children": [
                                            {
                                                "text": "Nested Lists B1"
                                            }
                                        ]
                                    },
                                    {
                                        "type": "numbered-list",
                                        "children": [
                                            {
                                                "type": "list-item",
                                                "children": [
                                                    {
                                                        "type": "list-item-text",
                                                        "children": [
                                                            {
                                                                "text": "Nested Lists B1a"
                                                            }
                                                        ]
                                                    }
                                                ]
                                            },
                                            {
                                                "type": "list-item",
                                                "children": [
                                                    {
                                                        "type": "list-item-text",
                                                        "children": [
                                                            {
                                                                "text": "Nested Lists B1b"
                                                            }
                                                        ]
                                                    }
                                                ]
                                            }
                                        ]
                                    }
                                ]
                            },
                            {
                                "type": "list-item",
                                "children": [
                                    {
                                        "type": "list-item-text",
                                        "children": [
                                            {
                                                "text": "Nested Lists B2"
                                            }
                                        ]
                                    }
                                ]
                            }
                        ]
                    }
                ]
            },
            {
                "type": "list-item",
                "children": [
                    {
                        "type": "list-item-text",
                        "children": [
                            {
                                "text": "Nested Lists C"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "type": "paragraph",
        "children": [
            {
                "text": ""
            }
        ]
    }
]
